UI/EnemyDetailsPanel.tscn

```diff
@@ -375,15 +375,42 @@ size_flags_horizontal = 3
 max_value = 4.29497e+09
 suffix = "XP"
 
-[node name="HSeparator" type="HSeparator" parent="VBoxContainer"]
+[node name="PpContainer" type="HFlowContainer" parent="VBoxContainer"]
 margin_top = 564.0
 margin_right = 1004.0
-margin_bottom = 568.0
+margin_bottom = 592.0
+hint_tooltip = "Named Params does not affect the play points dropped (yet)"
```

Comment:
Remove this, as it'll be outdated if/when the feature gets implemented

Reply:
This has been fixed.